src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  flex-direction: column;
+  display: flex;
+
+  &__info {
+    margin: 40px 16px 0;
+  }
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    padding-left: 19px;
+    padding-top: 32px;
+  }
+
+  &__info-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    font-style: normal;
+  }
+
+  &__info-code {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-top: 4px;
+    left: 16px;
+  }
+
+  &__info-review-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 166px;
+    height: 16px;
+    margin-top: 16px;
+
+  }
+
+  &__info-review-count {
+    font-size: 10px;
+    line-height: 14px;
+    font-style: normal;
+    font-weight: normal;
+    display: flex;
+    align-items: flex-end;
+  }
+
+  &__info-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    width: 166px;
+    height: 18px;
+    margin-top: 24px;
+  }
+
+  &__info-price-letter {
+    font-size: 12px;
+    line-height: 18px;
+    display: flex;
+    align-items: flex-end;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &__info-price-number {
+    font-size: 16px;
+    line-height: 18px;
+    font-style: normal;
+    font-weight: 700;
+    display: flex;
+    align-items: flex-end;
+    text-align: right;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    display: block;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 40px;
+    color: #fff;
+    text-decoration: none;
+    width: 166px;
+    height: 40px;
+    background-color: #00acdc;
+    border-radius: 5px;
+    border: 1px solid #00acdc;
+    margin: 16px;
+    left: 16px;
+    text-align: center;
+    text-transform: uppercase;
+  }
+
+  &__buy:hover {
+    background-color: rgb(255, 255, 255);
+    color: #00acdc;
+  }
```

Comment:
Check the 9th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

Reply:
⚠️ didn't fixed